app/routes/participants/profile.js

```diff
@@ -4,9 +4,9 @@ export default Ember.Route.extend({
     model(params) {
         return Ember.RSVP.hash(
             {
-                'profile': this.store.findRecord('profile', params.profile_id),
+                'profile': this.store.query('account', {'profiles.profileId': params.profile_id}),   // TODO: Requires globally unique profile IDs- format <acctShortId>.profileId
```

Comment:
That's basically what this does: `accountID.unique_part`. By "shortID", I'm referring to the account ID sans the `namespace.collection` prefix (which I assume would be totally redundant for all users, unless a profile was ever being referenced from a different namespace).

Let me know if that assumption is incorrect, eg if the `namespace.collection` part is necessary.
 
The main motivation was to make URLs of the form `/participants/profile/<profile_id>` slightly less verbose, while still making profile IDs unique.